dot/state/block_test.go

```diff
@@ -297,15 +297,26 @@ func TestLatestFinalizedRound(t *testing.T) {
 func TestFinalization_DeleteBlock(t *testing.T) {
 	bs := newTestBlockState(t, testGenesisHeader)
 	AddBlocksToState(t, bs, 5)
+
+	btBefore := bs.bt.DeepCopy()
+	t.Log(btBefore)
 	before := bs.bt.GetAllBlocks()
 	leaves := bs.Leaves()
 
+	// TODO: why isn't arrival time set?
```

Comment:
will open an issue for this, it's currently not used anywhere so it's ok to ignore for now i think